automatic/tightvnc/tightvnc.nuspec

```diff
@@ -29,8 +29,4 @@ With TightVNC, you can:
     <tags>remote control vnc admin</tags>
```

Comment:
This really should have tightvnc in the tags.
Also take a look at the [Contributing Guidelines draft](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guideliness-%5BDraft%5D#12-metadata) to see if any of the mandatory tags there apply